web_accounts_list.json

```diff
@@ -4263,6 +4263,62 @@
       "known_accounts" : ["summit1g","cohhcarnage"],
       "category" : "gaming",
       "valid" : false

```

Comment:
Does the pretty_uri change now make this site work? If so, lets change the "valid" to `true`

Reply:
Oh, I didn't realize that it was marked as not valid. I just spotted the inconsistency in the parameter name.

Reply:
thank you.